packages/manager/apps/web/client/app/hosting/multisite/git-association/hosting-multisite-git-association.html

```diff
@@ -0,0 +1,136 @@
+<oui-message data-type="error" data-ng-if="$ctrl.errorMessage">
+    <small data-ng-bind-html="$ctrl.errorMessage"></small>
+</oui-message>
+
+<h1 data-translate="hosting_multisite_git_association_title"></h1>
+<p data-translate="hosting_multisite_git_association_configure_description"></p>
+
+<form
+    class="form-inline"
+    name="$ctrl.gitAssociationForm"
+    data-ng-submit="$ctrl.generateLink()"
```

Comment:
```suggestion
    data-ng-submit="$ctrl.generateLink()"
```
I didn't find generateLink function in controller